### PR TITLE
Support Veres One type DIDs in the documentLoader.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**

### CODE REVIEW

File: lib/helpers.js [hunk 1]
```diff
@@ -311,11 +315,15 @@ function _getExpectedRootCapability({invocationTarget, edvId}) {
...
-// Note: for dereferencing `did:key` URLs
+// Note: for dereferencing `did:` URLs
 async function _documentLoader(url) {
-  // TODO: move to did-key lib
-  if(url.startsWith('did:key:')) {
-    const document = await didKeyDriver.get({url});
+  let document;
+  if(url.startsWith('did:')) {
+    document = await didIo.get({did: url});
+    // FIXME
```
Comment (mattcollier) [inline lib/helpers.js:323]: What is this FIXME about?

File: package.json [hunk 2]
```diff
@@ -25,31 +25,35 @@
...
-    "bedrock-passport": "4.0.2 - 5.x",
-    "bedrock-permission": "^2.5.0",
-    "bedrock-mongodb": "3.x - 6.x",
-    "bedrock-server": "^2.3.2",
-    "bedrock-validation": "^4.1.0",
+    "bedrock-passport": "^5.0.2",
+    "bedrock-permission": "^2.5.4",
+    "bedrock-mongodb": "^6.0.2",
+    "bedrock-server": "^2.6.0",
+    "bedrock-validation": "^4.3.0",
```
Comment (mattcollier) [inline package.json:49]: For my part I would rather you not use whatever tool you're using to change all this stuff.  Although I think these existing ranges have little basis in fact, I don't think it's cool to destroy them automatically like this.

Comment (dmitrizagidulin) [reply package.json:49]: No prob. Will revert. (I just suspect this is a cut&paste artifact).

File: lib/helpers.js (hunk 1 above)
Comment (dmitrizagidulin) [reply lib/helpers.js:323]: Just a reminder to replace lines 324-326 (since the return signature of `didIo.get()` will be changing.

Comment (dmitrizagidulin) [reply lib/helpers.js:324]: Not addressing this now (will address once did-io is refactored).

File: package.json (hunk 2 above)
Comment (gannan08) [reply package.json:49]: @dmitrizagidulin Releasing this package is blocked on your revert.

Comment (dlongley) [reply package.json:49]: No need to revert, please proceed.

**Pull request merged**